### ISSUE TITLE
Monthly reward for Market-Makers

### ISSUE THREAD

**Issue by alekkela** [rupaya-project/proposals#21]
3000 rupx

Monthly stipend to be given on the 1st of each month. 
This would pay out as 3000 rupx for month 1, 3000 rupx for month 2, 3000 rupx for month 3, 3000 rupx for month 4, 3000 rupx for month 5, 3000 rupx for month 6. 

Set to last for a period of 6mo. Then reviewed and put up for new proposal to either increase, keep the same, decrease, or cease stipend. If a new proposal is not put forward for renewal, then the monthly stipend will automatically cease.

This is to reward the makers that put up personal funds or time in the good faith and dedication to the Rupaya community.

This will help the community by creating an incentive to keep the market-makers actively engaged and feeling appreciated for their efforts. The funds will be used at the makers discretion, to help with soaking buys or freeing up funds to reinvest into the community. 

This is my first proposal
Alekkela#5444

**Comment by alekkela** [comment]
is it possible for this to be put up for a vote so that it can be considered for the upcoming month

**Comment by CryptoPoseidon987** [comment]
Considering that we love rewarding the community and the market makers most probably help a great deal in helping Rupaya and it's community stay on its feet, i feel like it would be nice initiative to reward the market makers. Now I'm not sure if the amount of rupx proposed is the right amount but I'm definitely liking the idea this proposal gives!

**Comment by mnzone-dev** [comment]
@alekkela thank you for opening the proposal for discussion.

I would appreciate if you could research a bit the legality of this market-making activity in the crypto world but also outside in traditional trading.

For me personally, Market-Making is a grey area that I normally stay away from.

**Comment by oooStrateg** [comment]
MM are doing good job so far, and they have a right to submit this proposal.
I have the only question, it's 3K / person or it's 3K pool?
Thanks

**Comment by oooStrateg** [comment]
@mnzone-dev 
We are on the unregulated market, there is nothing wrong with someone set up buy orders & sell orders.

**Comment by alekkela** [comment]
I have updated the verbiage and made it more friendly in so that we don't have to skirt legality issues. Also, I have updated the exact way the payouts will be received so that it is clear that it will be a per individual payout and not a pool.
 Please comment and let me know any suggestions for wording or otherwise you may have. I believe in constructive criticism, so please be helpful if you can be

**Comment by 001Crypto001** [comment]
Hi Alekkela, I liked your idea in having a stable market for RUPX but is this consider legal to have someone manipulating the market to favour our coin...
And regarding the 3000 RUPX rewards per month, is there any specific reason for that amount...
Or maybe can plan to have 1 dedicated MN for this type of activities and just shower the rewards back to the community for holding RUPX for certain period...

**Comment by pitter98** [comment]
@001Crypto001 
While I agree it may be a gray area, as @oooStrateg said, it is a completely unregulated market, so there really are no legality issues.

With that being said, I think it needs to be made clear that the market making is definitely not being done to manipulate the price.  The market makers are only trying to help to keep a stable price (though lately have been facing some serious headwinds), and it is not a pump and dump style tactic or anything like that.  The market makers try to keep a steady amount of buy and sell orders to try to keep the price from crashing or rising too fast.

On the reward amount, I can't say whether others will think it is too high, too low, or good, but I think it seems like a fair amount if there is active participation all month long.

**Comment by oooStrateg** [comment]
@alekkela
_Why you request 126K?_ It's like a stipend for 7 people. ( 7x3Kx6M)
If you represent some others members please mention who they are, if you are alone it's not mentioned who is escrow or beneficiary and how funds would be managed. 
With 100K we can develop/purchase MM - AI
Olso I don't see any particular reasons for 6mo sign up.
And we can't risk treasury abuse so, please think twice measure pro & cons and submit something more realistic.
For me, it's strong  **NO** in the current state.

**Comment by alekkela** [comment]
It is for 7 people and it would be a fund managed by mobay and velk. This would be distributed to us on the 1st of the month by them...we would never see the fund ourselves.

**Comment by alekkela** [comment]
The name of the current consultants have been added to the proposal. Who will receive the funds for distribution to the consultants has also been clarified. The dispersal location of the funds, to the consultants, has also been updated

**Comment by Gambobit** [comment]
I agree on the idea at least the following 6 months it makes sense. I would like that the contributors be rewarded regularly but selected through voting (could be by just the team) but this to ensure that someone not listed in here has incentives to contribute or if by some reason someone from the list struggles to contribute in a month gets unfairly rewarded

**Comment by alekkela** [comment]
I selected a longer term projection because proposals can only be started on new superblocks.  As far as voting goes, that is fine for the future, but given the current group was selected by mobay and has already been doing work for the last 2 mo, i think it would be unfair to start fresh. Remember, they did this when nobody else wanted to and without compensation. Thats why I also limited the stipend link to 6 mo. as it would allow for changes to be made bi-annually, if changes were needed.

**Comment by alekkela** [comment]
updated to reflect the current consultants that have updated their status. Rupx has been adjusted accordingly to reflect the amount of consultants to be paid, 1 person pending, will update again
